Tools/BiblioTech/Commands/CheckCidCommand.cs

```diff
@@ -32,7 +39,78 @@ protected override async Task Invoke(CommandContext context)
 
             var response = await checker.PerformCheck(cid);
             await Program.AdminChecker.SendInAdminChannel($"User {Mention(user)} used '/{Name}' for cid '{cid}'. Lookup-success: {response.Success}. Message: '{response.Message}' Error: '{response.Error}'");
+
+            if (response.Success)
+            {
```

Comment:
Plz pull everything inside this if-case to its own function. Probably something like CheckAltruisticRole() maybe?